docs/index.md

```diff
@@ -28,6 +28,7 @@ After that, you need to instrument your tests using the **Scope agent** for your
 
 * [.NET agent instructions](dotnet-installation.md)
 * [Java agent instructions](java-installation.md)
+* [Scala agent instructions](java-installation.md)
```

Comment:
I wouldn't add the Scala agent as a different one. New Relic, Datadog, etc. just add Scala as a supported feature of their Java agent. I'd do the same.

Reply:
Oki. Thanks!